Lib/test/test_asyncio/test_tasks.py

```diff
@@ -2664,6 +2668,7 @@ class CTaskSubclass_PyFuture_Tests(BaseTaskTests, test_utils.TestCase):
 
     Task = getattr(tasks, '_CTask', None)
     Future = futures._PyFuture
+    all_tasks = getattr(tasks, '_py_all_tasks', None)
```

Comment:
did you intend this to be `_c_all_tasks`? (to match the `Task` implementation)

Reply:
Yes, fixed.